test/fixtures/critical-styles/assets/global.css

```
@@ -0,0 +1,9 @@
+.global-used {
+  --global-critical: present;
+  color: #ff0000;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)